store/employee/mutations.ts

```diff
@@ -1,14 +1,10 @@
-import {MutationTree} from 'vuex';
+import {MutationTree} from 'vuex/types';
```

Comment:
Importing from "/types" has the same effect as this 👇?

```suggestion
import type {MutationTree} from 'vuex/types';
```